src/Tizen.NUI/src/public/BaseComponents/ViewEvent.cs

```diff
@@ -1364,5 +1364,25 @@ private void OnWindowWheelEvent(object sender, Window.WheelEventArgs e)
             }
         }
 
+        /// <summary>
+        /// TouchArea can reset the view's touchable area.<br/>
+        /// If you set the TouchArea on an view, when you touch the view, the touch area is used rather than the size of the view.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Size2D TouchArea
+        {
+            get
+            {
+                Size2D temp = new Size2D(0, 0);
+                GetProperty(View.Property.TouchArea).Get(temp);
+                return new Size2D(temp.Width, temp.Height);
```

Comment:
are these different?
`return new Size2D(temp.Width, ...)` and `return temp` ?
if they are same, I think the latter would be better. 
How do you think?